addon/components/sortable-group.js

```diff
@@ -18,29 +25,61 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get: function() {
+      return a(); 
+    },
+    set: function(_, value){
+     return a(value);
+    }
+  }), 
   /**
-    Vertical position for the first item.
-
-    @property itemPosition
-    @type Number
+    @method _getFirstItemPosition
+    @private
   */
-  itemPosition: computed(function() {
+  _getFirstItemPosition: function(direction) {
     let element = this.element;
     let stooge = $('<span style="position: absolute" />');
-    let result = stooge.prependTo(element).position().top;
+    let result;
+    if(direction === 'y') {
```

Comment:
This is really nitpicky but could we have a space between `if` and `(`? Sorry, old habits on my part!